### PR TITLE
Add 0.13.0 changelog and bump version to 0.13.0

### PR DESCRIPTION
**Pull request by rocodes**
## Status

Ready for review 

## Description

Refs #2172 

## Test Plan
- [x] Base branch is release/0.13.0
- [x] Visual review + CI

## Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:

 - [ ] I have tested these changes in the appropriate Qubes environment
 - [ ] I do not have an appropriate Qubes OS workstation set up (the reviewer will need to test these changes)
 - [ ] These changes should not need testing in Qubes

If these changes add or remove files other than client code, the AppArmor profile may need to be updated. Please check as applicable:

 - [ ] I have updated the [AppArmor profile](https://github.com/freedomofpress/securedrop-client/blob/HEAD/files/usr.bin.securedrop-client)
 - [x] No update to the AppArmor profile is required for these changes
 - [ ] I don't know and would appreciate guidance

If these changes modify the database schema, you should include a database migration. Please check as applicable:

 - [ ] I have written a migration and upgraded a test database based on `main` and confirmed that the migration is [self-contained] and applies cleanly
 - [ ] I have written a migration but have not upgraded a test database based on `main` and would like the reviewer to do so
 - [ ] I need help writing a database migration
 - [x] No database schema changes are needed

[self-contained]: https://github.com/freedomofpress/securedrop-client#generating-and-running-database-migrations

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 0.13.0
+
+* Broaden printing support to all filetypes supported by LibreOffice (#2166)
+* Display an error message if user tries to print an unprintable file type (#2166)
+* Use magic bytes to detect file type in print workflow (#2166)
+* Use single PrintDialog GUI element for all print actions (#2143)
+* Remove unused _from_string methods from SDK (#2144)
+* Update build requirements (#2136) and a development dependency (#2138)
```
Comment (legoktm) [inline changelog.md:10]: Can we be more specific here?

```suggestion
* Update build toolchain and `setuptools` for CVE-2024-6345  (#2136, [securedrop-builder#501](https://github.com/freedomofpress/securedrop-builder/pull/501)) 
* Update `ruff` to 0.5.4 (#2138)
```

**Review by legoktm — COMMENTED** [review]
See suggestion - also missing version bump?

**Review by legoktm — DISMISSED** [review]
LGTM, pending CI.

**Comment by rocodes** [comment]
(Sorry, it needed one more push - the `debian/changelog` file was wrong)

**Review by legoktm — APPROVED** [review]
Ooh, good catch.

**Pull request merged**